packages/grafana-ui/src/components/ClipboardButton/ClipboardButton.tsx

```diff
@@ -2,41 +2,25 @@ import React, { useCallback, useRef } from 'react';
 
 import { Button, ButtonProps } from '../Button';
 
-/** @deprecated Will be removed in next major release */
-interface ClipboardEvent {
-  action: string;
-  text: string;
-  trigger: Element;
-  clearSelection(): void;
-}
-
 export interface Props extends ButtonProps {
   /** A function that returns text to be copied */
   getText(): string;
   /** Callback when the text has been successfully copied */
-  onClipboardCopy?(e: ClipboardEvent): void;
+  onClipboardCopy?(copiedText: string): void;
   /** Callback when there was an error copying the text */
-  onClipboardError?(e: ClipboardEvent): void;
+  onClipboardError?(copiedText: string): void;
 }
 
-const dummyClearFunc = () => {};
-
 export function ClipboardButton({ onClipboardCopy, onClipboardError, children, getText, ...buttonProps }: Props) {
   const buttonRef = useRef<null | HTMLButtonElement>(null);
   const copyTextCallback = useCallback(async () => {
     const textToCopy = getText();
-    // Can be removed in 9.x
-    const dummyEvent: ClipboardEvent = {
-      action: 'copy',
-      clearSelection: dummyClearFunc,
-      text: textToCopy,
-      trigger: buttonRef.current!,
-    };
+
     try {
       await copyText(textToCopy, buttonRef);
-      onClipboardCopy?.(dummyEvent);
+      onClipboardCopy?.(textToCopy);
     } catch {
-      onClipboardError?.(dummyEvent);
+      onClipboardError?.(textToCopy);
```

Comment:
While we're here, should maybe we send through the error?

Reply:
Ah, yeah, that's a good point.
Reckon something like this would be okay?
```ts
onClipboardError?(copiedText: string, error: unknown)
```

Not exactly sure what the interface of the error is

Reply:
PERFECT 🚀